src/codewhisperer/util/authUtil.ts

```diff
@@ -72,7 +72,10 @@ export class AuthUtil {
                 vscode.commands.executeCommand('aws.codeWhisperer.refreshStatusBar'),
                 vscode.commands.executeCommand('aws.codeWhisperer.updateReferenceLog'),
             ])
-            await vscode.commands.executeCommand('aws.codeWhisperer.enableCodeSuggestions')
+            // To check valid connection
+            if (this.isValidEnterpriseSsoInUse() || this.isBuilderIdInUse()) {
+                await vscode.commands.executeCommand('aws.codeWhisperer.enableCodeSuggestions')
```

Comment:
This looks like an important change but how is it related to the "getting started ux"? Should this be in a separate PR, possibly with a changelog item (unless it's fixing code that wasn't released yet)?

Reply:
This is fixing code that was not release yet.

User should show Learn UX only If user is logged in

Reply:
This checks 
If users if logged In either by builderId or EnterpriseSso and not has expired connection then user is able to see the Learn UX

Reply:
how is `enableCodeSuggestions` related to showing the Learn UX??

Reply:
We are checking whether user is logged in in enable suggestions.